Attendee and Staff Scanner menu update

nathannwangg opened the pull request:
## Description

<!--

- Added and updated background images for attendee and staff scanner menus
- Added new colors to HIAppearance.swift for updated scanner buttons that match the new 2025 designs. Replaced old definitions of scannerButton colors with the new colors
- Edited the definition of QRSelection style in HILabel.swift to change the scanner button text colors to white, replacing last year's brown
- Updated the colors for all attendee scanner buttons and changed the view title color to white
- Updated the colors for all staff scanner buttons and changed the view title style to viewTitle instead of viewTitleBrown

-->

## Screenshots

<!--
<img width="421" alt="image" src="https://github.com/user-attachments/assets/1a1a29ca-4288-4907-8722-2e5814fc35ba" />
![image](https://github.com/user-attachments/assets/b61d9e93-e648-4f07-83b1-b2802a4fab8a)
-->

anushkasankaran commented:
Great job! Only thing is I think the Points Shop button is available for staff, not attendees.

nathannwangg commented:
Just moved the points shop scanner button to the staff scanner selection menu and removed it from the attendee scanner selection menu.

Screenshots:

<img width="394" alt="image" src="https://github.com/user-attachments/assets/6a12f68e-fda5-4053-8fed-7e1b9d965cc5" />

![image](https://github.com/user-attachments/assets/2efe4404-4edd-40c0-b780-b2a38ed6374c)
